Do not install u-deva, aspell-hi ships the same charset.

diff -Nrup a/Makefile.pre b/Makefile.pre
--- a/Makefile.pre	2005-05-24 12:57:48.000000000 +0200
+++ b/Makefile.pre	2026-10-22 17:22:14.878213741 +0200
@@ -4,7 +4,7 @@ lang = mr
 version = 0.10-0
 
 cwl_files = mr.cwl
-data_files = mr.dat u-deva.cset u-deva.cmap
+data_files = mr.dat
 doc_files = README COPYING Copyright
 extra_files = configure info Makefile.pre
 multi_files = marathi.alias mr.multi
